src/Illuminate/Encryption/OpenSslEncrypter.php

```diff
@@ -77,6 +95,7 @@ public static function generateKey($cipher)
      * @param  bool  $serialize
      * @return string
      *
+     * @throws \Exception
```

Comment:
?

Reply:
My IDE was complaining about `random_bytes()` throwing it

Reply:
Ok, I removed `@throws \Exception` from there, `OpenSslEncrypter::encrypt()`, and `EncryptionManager::generateKey()` doc blocks